### PR TITLE
Add localization support for groups

### LINKED ISSUE #3558: Groups should support multilingual schema fields
Group templates are missing translation support for multilingual fields. E.g. templates should call get_translated for displayed group fields. I'll provide suggestions with a PR in a minute.

### PR DESCRIPTION
**Pull request by mikkonieminen**
# Fixes
#3558 

### Proposed fixes:

Fixes missing support for multilingual group fields in group templates.

### Features:

- [ ] includes tests covering changes
- [ ] includes updated documentation
- [x] includes user-visible changes
- [ ] includes API changes
- [ ] includes bugfix for possible backport

Please [X] all the boxes above that apply

### CODE REVIEW

**Comment by wardi** [comment]
@mikkonieminen thanks! would you take a look at the test failures?

**Comment by mikkonieminen** [comment]
@wardi: fixed

File: ckan/templates/group/about.html [hunk 1]
```diff
@@ -3,14 +3,22 @@
 {% block subtitle %}{{ _('About') }} - {{ super() }}{% endblock %}
 
 {% block primary_content_inner %}
-  <h1>{% block page_heading %}{{ c.group_dict.display_name }}{% endblock %}</h1>
+  <h1>
+    {% block page_heading %}
+      {% if c.group_dict.title %}
```
Comment (wardi) [inline ckan/templates/group/about.html]: this doesn't look right. The translated version will be stored in `title_translated` so checking the title relies on the correct validator being set up for `title`. Why don't you follow the `h.get_translated(c.group_dict, 'title') or c.group_dict.display_name` pattern from below instead?

File: ckan/templates/group/snippets/info.html [hunk 2]
```diff
@@ -11,16 +11,20 @@
...
         [{{ _('Deleted') }}]
       {% endif %}
     </h1>
     {% endblock %}
     {% block description %}
-    {% if group.description %}
+    {% if h.get_translated(group, 'description') %}
       <p>
-        {{ h.markdown_extract(group.description, 180) }}
+        {{ h.markdown_extract(h.get_translated(group, 'description'), 180) }}
```
Comment (wardi) [inline ckan/templates/group/snippets/info.html]: you're not extracting the description from the current language here

File: ckan/templates/group/about.html (hunk 1 above)
Comment (mikkonieminen) [inline ckan/templates/group/about.html]: True, had the same idea before but forgot to actually do the fix. I'll commit the same fix to the other remaining templates in a moment.

**Comment by mikkonieminen** [comment]
For the localization to actually work in group index the include_extras needs to be set to true for the group_list action call in GroupController. Is this a valid solution performance wise?

**Pull request merged**